modules/object_in_world.py

```diff
@@ -38,3 +42,4 @@ def __init__(
         self.location_x = location_x
         self.location_y = location_y
         self.spherical_variance = spherical_variance
+        self.object_type = object_type
```

Comment:
Make this `label` and of type integer following convention (e.g. `detections_and_time` , `detection_in_world` ).